modules/javafx.base/src/main/java/javafx/beans/Observable.java

```diff
@@ -93,4 +94,34 @@ public interface Observable {
      */
     void removeListener(InvalidationListener listener);
 
+    /**
+     * Creates a {@link Subscription} on this {@code Observable} which calls
+     * {@code invalidationSubscriber} whenever it becomes invalid. If the same
+     * subscriber is subscribed more than once, then it will be notified more
+     * than once. That is, no check is made to ensure uniqueness.
+     * <p>
+     * Note that the same subscriber instance may be safely subscribed for
+     * different {@code Observables}.
+     * <p>
+     * Also note that when subscribing on an {@code Observable} with a longer
+     * lifecycle than the subscriber, the subscriber must be unsubscribed
+     * when no longer needed as the subscription will otherwise keep the subscriber
+     * from being garbage collected.
```

Comment:
I think that the explanations starting from "If the same" are no longer needed if you add something like what I suggested for `Subscription`. It covers the lifecycle and multiple subscriptions explanations already.